rbx_reflector/src/cli/defaults_place.rs

```diff
@@ -63,6 +63,8 @@ fn save_place_in_studio(path: &PathBuf) -> anyhow::Result<StudioInfo> {
     log::info!("Starting Roblox Studio...");
 
     let mut studio_process = Command::new(studio_install.application_path())
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
```

Comment:
I've never seen Roblox Studio emit any useful logs or errors, so I think this is a good change. I think it's possible we may eventually want to see what Roblox Studio outputs, but we can decide what to do about that another day.